source/_patterns/00-base/05-colors/colors.json

```diff
@@ -2,61 +2,62 @@
   "items": [
     {
       "name": "$c-red",
-      "value": "hsl(352, 92%, 38%)",
-      "comment": "Primary Color. Used for links"
```

Comment:
Our new sass to json blows away comments. They weren't absolutely essential, but maybe we can work out some other way to bring them back.